templates/devspace.yaml.tpl

```diff
@@ -332,6 +335,17 @@ profiles:
             labelSelector: 
               app: ${DEVENV_DEPLOY_APPNAME}
 
+  - name: onlyForwardDlv
```

Comment:
Should another profile for ForwardAllPorts mode be added?

Reply:
From the perspective of devspace yaml the "default" is to forward all ports. However, by default the devenv code sets DEVENV_DEV_ONLY_FORWARD_DELVE=true, so from the end user perspective, only forwarding the delve port is the default.

I think we can avoid another profile because of that